modules/mono/mono_gd/gd_mono_utils.cpp

```diff
@@ -667,7 +663,6 @@ void print_unhandled_exception(MonoException *p_exc) {
 }
 
 void set_pending_exception(MonoException *p_exc) {
-#ifdef HAS_PENDING_EXCEPTIONS
```

Comment:
Looks like Travis' mono doesn't like this: https://travis-ci.org/godotengine/godot/jobs/554433349

Reply:
Seems like we're still running 5.10.1.20 from the debian wheezy upstream repo on Travis:
```
Get:2 http://download.mono-project.com/repo/debian wheezy/main amd64 mono-runtime-sgen amd64 5.10.1.20-0xamarin1+debian7b1 [1,760 kB]
```
I don't know if there's a more recent repo compatible with Travis' Ubuntu Xenial.

Reply:
Yeah the `mono` source is `wheezy`:
https://github.com/travis-ci/apt-source-safelist/blob/0c2267bff66def573d31a8240ff7d31795e1f6fa/ubuntu.json#L418-L423

I'll see if they could add http://download.mono-project.com/repo/debian/dists/stable-xenial/ to the whitelist, but I don't have high hopes of Travis acting fast, so we should likely keep this define for now.

Reply:
Well there's my own year old issue: https://github.com/travis-ci/apt-source-safelist/issues/384

Reply:
Added a travis workaround:

```
is_travis = os.environ.get('TRAVIS') == 'true'

if is_travis:
    # Travis CI may have a Mono version lower than 5.12
    env_mono.Append(CPPDEFINES=['NO_PENDING_EXCEPTIONS'])
```

Reply:
Sounds good. So for non-Travis we would now require a version >= 5.12 I suppose?

Reply:
We've had that requirement for a year :P https://github.com/godotengine/godot-docs/blame/82963cf7055bad24694d88d3ecc6a4da04f14fa7/development/compiling/compiling_with_mono.rst#L11